File: 6/go/main.go
package main

func solve(limit int) int {
	var squared_sum int = 0
	var sum_squared int = 0
	for i := 0; i <= limit; i++ {
		squared_sum += i * i
		sum_squared += i
	}
	sum_squared = sum_squared * sum_squared
	var difference = sum_squared - squared_sum
	return difference
}

func main() {
	println(solve(100))
}

File: 6/python/main.py
def solve(limit: int) -> int:
    squared_sum = 0
    sum_squared = 0
    for i in range(1, limit + 1):
        squared_sum += i**2
        sum_squared += i
    sum_squared = sum_squared**2
    difference = sum_squared - squared_sum
    return difference


if __name__ == '__main__':
    print(solve(100))
